### ISSUE TITLE
help page accessibility issues

### ISSUE THREAD

**Issue by SNefertitiB** [gcivil-nyu-org/team-1-inperson#206]
- VoiceOver doesn't read out the help page information

**Comment by claudiaashao** [comment]
this is really weird considering how the help page is just plain text :( I can try with firefox
